enos/ansible/roles/backup/tasks/rally.yml

```diff
@@ -3,10 +3,15 @@
   stat: path=/root/rally_home
   register: conf
 
+#- name: Make a tar of the rally execution environment
+#  archive:
+#    path: /root/rally_home
+#    dest: /root/rally.tar.gz
+# NOTE(msimonin): There are symlink in RALLY_HOME which
+# cause some trouble to ansible archive module
+# https://github.com/ansible/ansible/issues/24427
 - name: Make a tar of the rally execution environment
-  archive:
-    path: /root/rally_home
-    dest: /root/rally.tar.gz
+  command: tar -czf /root/rally.tar.gz /root/rally_home
```

Comment:
Actually we can't use `-h` because the symlink is created in the docker container and reference a path inside that container, which correspond to nothing in the host machine.
Using `-h` makes the task fail.